file.js

```diff
@@ -5,6 +5,7 @@ const { once } = require('events')
 
 module.exports = async function (opts = {}) {
   const destOpts = { dest: opts.destination || 1, sync: false }
+  if (opts.append === false) destOpts.append = false
   if (opts.mkdir) destOpts.mkdir = true
   const destination = pino.destination(destOpts)
```

Comment:
Would this not be more comprehensive?

```js
const destOpts = Object.assign({}, opts.destination, { dest: opts.destination || 1, sync: false })
```

Reply:
That's certainly a valid approach, but it would expand the scope of this issue. If you prefer to forward all the options to the destination, I won't object. @mcollina, where do you stand on this?

Reply:
go for it, it's better